realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -98,6 +100,19 @@ public void onError(SyncSession session, ObjectServerError error) {
     private static volatile AuthenticationServer authServer = new OkHttpAuthenticationServer();
     private static volatile UserStore userStore;
 
+    private static NetworkStateReceiver.ConnectionListener networkListener = new NetworkStateReceiver.ConnectionListener() {
+        @Override
+        public void onChange(boolean connectionAvailable) {
+            if (connectionAvailable) {
+                RealmLog.debug("networkListener connection available");
```

Comment:
Perhaps: `NetworkListener: Connection available` and `NetworkListener: Connection lost` for better readability.